crates/oxc_linter/src/snapshots/eslint_no_multi_assign.snap

```diff
@@ -0,0 +1,138 @@
+---
+source: crates/oxc_linter/src/tester.rs
+snapshot_kind: text
+---
+  ⚠ eslint(no-multi-assign): Do not use chained assignment
+   ╭─[no_multi_assign.tsx:1:9]
+ 1 │ var a = b = c;
+   ·         ─────
+   ╰────
```

Comment:
can we add a help describing either why this is bad or how to fix?


Consider moving this into two separate assignments... or something similar

Reply:
Thanks, updated in cb17eb5